src/hostname/hostnamed.c

```diff
@@ -369,6 +428,65 @@ static int property_get_hostname(
         return sd_bus_message_append(reply, "s", current);
 }
 
+static int property_get_static_hostname(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        Context *c = userdata;
+        assert(c);
+
+        context_read_etc_hostname(c);
+
+        return sd_bus_message_append(reply, "s", c->data[PROP_STATIC_HOSTNAME]);
+}
+
+static int property_get_machine_info_field(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        sd_bus_slot *slot;
+        Context *c;
+
+        /* The the current context without the property userdata offset added */
```

Comment:
"The the". Dunno, even with the comment, it took me a moment to figure out what is going on here. Maybe extend the comment a bit?